src/ACadSharp/Entities/Spline.cs

```diff
@@ -112,7 +114,82 @@ public Spline() : base() { }
 		/// <inheritdoc/>
 		public override BoundingBox GetBoundingBox()
 		{
-			return BoundingBox.FromPoints(this.ControlPoints);
+			List<XYZ> vertices = this.PolygonalVertexes(256);
+
+			return BoundingBox.FromPoints(vertices);
+
+			// previous implementation: control points can be outside the bounding box of the spline curve
+			//return BoundingBox.FromPoints(this.ControlPoints);
+		}
+
+		/// <summary>
+		/// Converts the spline in a list of vertexes.
+		/// </summary>
+		/// <param name="precision">Number of vertexes generated.</param>
+		/// <returns>A list vertexes that represents the spline expressed in object coordinate system.</returns>
+		public List<XYZ> PolygonalVertexes(int precision)
+		{
+			if (precision < 2)
+			{
+				throw new ArgumentOutOfRangeException(nameof(precision), precision, "The precision must be equal or greater than two.");
+			}
+
+			List<XYZ> ocsVertexes = new List<XYZ>();
+			for (int i = 0; i < precision; i++)
+			{
+				double t = (double)i / (double)(precision-1);
+				ocsVertexes.Add(this.PointOnSpline(t));
+			}
+
+
+				return ocsVertexes;
+		}
+
+		/// <summary>
+		/// Gets a point on the spline.
+		/// </summary>
+		/// <param name="t">Parametric position on spline, between 0 and 1 (not linearly related to curve length).</param>
+		/// <returns>Point coordinates for the given parametric position on the spline.</returns>
+		public XYZ PointOnSpline(double t)
```

Comment:
Same as `PolygonalVertexes()`, it would be good to have a test for this one too.